content/yaml-code-signing/code-signing-identities.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Code signing identities
+description: How to set up code signing identities
+weight: 4
+aliases: /code-signing-yaml/code-signing-identities
+---
+
+Teams can use code signing identities to manage their code signing files in **Team settings > codemagic.yaml settings > Code signing identities** and reference them from their `codemagic.yaml` configuration.
```

Comment:
Add another sentence of what the benefit of this approach is (e.g having one place where code signing files are managed instead of doing it per each app).